### PR TITLE
🐛 Restore `qt: 'standard'` to SolrService

### PR DESCRIPTION
**Pull request by jeremyf**
**tl;dr** Can anyone explain why we don't want `qt: 'standard'` for Valkyrie connections?

In `ActiveFedora::SolrService` we always assign the `qt: 'standard'`.[1]

However, with [this old commit][1] we dropped passing the `qt: 'standard'` when we had `use_valkyrie` of true.

We discovered this bug when upgrading Hyku and Bulkrax to Hyrax v3.x and some.

Closes https://github.com/samvera/hyrax/issues/6676

Related to:

- https://github.com/samvera/hyrax/pull/3857

[1]:https://github.com/samvera/active_fedora/blob/8e7d365a087974b4ff9b9217f792c0c049789de6/lib/active_fedora/solr_service.rb#L40-L48
[2]:https://github.com/samvera/hyrax/commit/be6104fd13117f6a6c5b0252acd5b341247f04d3

@samvera/hyrax-code-reviewers

### CODE REVIEW

**Comment by dlpierce** [comment]
Perhaps it was to allow the application more flexibility when making solr queries?
I realize it was many moons ago, but perhaps @hackartisan has some idea?

**Comment by jeremyf** [comment]
@dlpierce The main problem is that it means `ActiveFedora::SolrService` and `Hyrax::SolrService` are not interchangeable; which I assume will mean that even though the `Hyrax::SolrService` might fall back to the `ActiveFedora::SolrService` it won't be the same types of queries.  Thus introducing breaks for those that are migrating their Hyrax application.

I have an amendment in mind that should be adequate for allowing both.  It'll take a bit.

**Comment by hackartisan** [comment]
Yeah, sorry I have specific memory about that param!
